AI/combine_set.py

```diff
@@ -5,29 +5,29 @@
 from sklearn.preprocessing import LabelEncoder
 
 
-def findClothes(category):
+def findClothes(category, data):
     bottom_items = set()
     shoes_items = set()
 
     # znajdź góry
-    top_items = set([tuple(row) for row in data[1:] if row[1] == "top"])
+    top_items = set([tuple(row) for row in data[1:] if "Upper garment" in row])
     # wylosuj jedną
     top_choice = random.sample(list(top_items), 1)[0]
     # zapisz jej kolor
-    color = top_choice[3]
+    color = top_choice[8]
 
     # jeśli wybrano zestaw monochromatyczny
     match category:
         case "one_color":
             bottom_items = [
-                row for row in data[1:] if row[3] == color and row[1] == "bottom"
```

Comment:
Podając z obiektu SQLAlchemy zmieniamy z dict na list - a to czasami miesza indexy - stąd sprawdzane jest czy dany typ ogólnie znajduje się w rekordzie